BlazarUI/app/scripts/components/repo-build/InterProjectAlert.jsx

```diff
@@ -56,6 +56,23 @@ class InterProjectAlert extends Component {
         </li>
       );
     });
+
+    return <ul>{listItemNodes}</ul>;
+  }
+
+  renderFailedBuildDetails() {
+    const {failedRepoBuilds} = this.props.upAndDownstreamModules;
+
+    if (Object.keys(failedRepoBuilds).length === 0) {
```

Comment:
For future reference, I think most people would just do
```
if (!Object.keys(failedRepoBuilds).length)
```
https://github.com/HubSpot/javascript#comparison--shortcuts